admin/services/jwt.service.js

```diff
@@ -12,21 +13,24 @@ const jwtService = {
   /**
    *
    * @param pupil
+   * @param checkWindowEndDate
```

Comment:
What type of object is this?   Please document.

Reply:
@jon-shipley updated